crud/common/utils.lua

```diff
@@ -199,18 +199,78 @@ function utils.tarantool_supports_uuids()
     return enabled_tarantool_features.uuids
 end
 
-function utils.convert_operations(user_operations, space_format)
+local function exists_operations_id_map(operations)
+    local map = {}
+    for _, operation in ipairs(operations) do
+        map[operation[2]] = true
+    end
+
+    return map
+end
+
+local function add_nullable_fields_rec(operations, operations_id_map, space_format, tuple, id)
+    if id < 2 or tuple[id - 1] ~= box.NULL then
+        return operations
+    end
+
+    if space_format[id - 1].is_nullable and not operations_id_map[id - 1] then
+        table.insert(operations, {'=', id - 1, box.NULL})
```

Comment:
maybe it's possible to use table.insert(operations, index, {'=', id - 1, box.NULL}) here to eliminate table.sort below?